### PR TITLE
LPS-190850 Adapt the Pagespeed Insights tab to the new design

### PR DESCRIPTION
**Pull request by veroglez**
https://liferay.atlassian.net/browse/LPS-190850

This pull request depends on:

https://github.com/liferay-echo/liferay-portal/pull/12945
https://github.com/liferay-echo/liferay-portal/pull/12920

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by veroglez** [comment]
ci:test:sf

File: modules/apps/layout/layout-reports-web/src/main/resources/META-INF/resources/js/components/render_times/FragmentList.tsx [hunk 1]
```diff
@@ -170,9 +170,10 @@ export default function FragmentList({
 								</span>
 
 								<span>
-									{renderTime}
-
-									{Liferay.Language.get('ms')}
+									{sub(
+										Liferay.Language.get('x-ms'),
+										renderTime
+									)}
```
Comment (victorg1991) [inline modules/apps/layout/layout-reports-web/src/main/resources/META-INF/resources/js/components/render_times/FragmentList.tsx:176]: This is super nice :D

**Comment by victorg1991** [comment]
ci:test:relevant

**Comment by ealonso** [comment]
ci:forward:force

**Pull request closed without merging**